### DISCUSSION TITLE
Adding Wikipedia articles

### DISCUSSION THREAD

**Discussion by kiug** [monkeytypegame/monkeytype#4478] [Ideas] [answered]
What do you think about adding Wikipedia articles?

**Comment by Miodec** [comment, marked as answer]
Settings > Funbox > Wikipedia.

**Comment by kiug** [comment]
Thank You.

I can't enable this mode.
I have this message: "You can't enable wikipedia with currently active config."

**Reply by Miodec** [reply]
You need to be in time or words mode.

**Reply by kiug** [reply]
It works.
There are articles only from English wikipedia?

**Reply by Miodec** [reply]
It shoould support other languages, but im not exactly sure about the list of possible languages.

**Reply by kiug** [reply]
Language list is in this file?
[https://github.com/monkeytypegame/monkeytype/blob/master/frontend/src/ts/test/wikipedia.ts](https://github.com/monkeytypegame/monkeytype/blob/master/frontend/src/ts/test/wikipedia.ts)

It is any chance, to add Polish language?

**Reply by Miodec** [reply]
Ill have a look

**Reply by kiug** [reply]
Thanks a lot :)

**Reply by Miodec** [reply]
Should be live now.

**Reply by kiug** [reply]
Yes it is, thanks a lot again.
